bio/quast/wrapper.py

```
@@ -4,11 +4,15 @@
 
 
 import os
+import tempfile
+from pathlib import Path
```

Ruff

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)